examples/gears/vulkangear.cpp

```diff
@@ -53,13 +53,13 @@ void VulkanGear::generate(GearInfo *gearinfo, VkQueue queue)
 	r0 = gearinfo->innerRadius;
 	r1 = gearinfo->outerRadius - gearinfo->toothDepth / 2.0f;
 	r2 = gearinfo->outerRadius + gearinfo->toothDepth / 2.0f;
-	da = 2.0f * M_PI / gearinfo->numTeeth / 4.0f;
+	da = static_cast<float>(2.0f * M_PI / gearinfo->numTeeth / 4.0f);
```

Comment:
For consideration: When using C++20 this and other places could be using `std::numbers::pi_v<float>` instead of the `M_PI` which is a `#define` and considered somewhat obsolete. Also, with `std::numbers::pi_v<float>` the cast would not be needed.

Reply:
I don't plan on using C++ 20. The samples support a wide range of platforms incl. some not widely used outside of some niches. And some of those lack behind with newer C++ features.